src/components/pages/Offerers/Offerer/VenueV1/fields/AccessibilityFields/AccessibilityFields.jsx

```diff
@@ -21,34 +21,34 @@ const checkNoDisabilityCompliant = (_value, allValues) => {
   return false
 }
 
+const initializeEmptyValue = fieldName => (_value, allValues) => {
+  return allValues[fieldName] === undefined ? false : allValues[fieldName]
+}
+
+const setNoDisabilityCompliance = fieldName => (value, allValues) => {
+  if (value) return false
+  return allValues[fieldName]
+}
```

Comment:
pour rendre plus claire ce que fait la fonction et à quoi 'value' fait référence : 
c'est pas immédiatement clair si on ne comprend pas le fonctionnement des décorateurs

```suggestion
const setNoDisabilityComplianceFor = fieldName => (noDisabilityComplianceValue, allValues) => noDisabilityComplianceValue ? false : allValues[fieldName]
```

Reply:
J'aime bien garder la signature avec les argument comme sur la doc, c'est plus facile à comprendre à la relecture du code + doc.